bin/p2-bootstrap/bootstrap.go

```diff
@@ -102,7 +103,9 @@ func VerifyConsulUp(timeout string) error {
 		return nil
 	}
 
-	store := kp.NewStore(kp.Options{})
+	store := kp.NewStore(kp.Options{
+		Token: *consulToken, // not actually necessary because this endpoint is unauthenticated
```

Comment:
The comment isn't necessarily true - different organizations may point to an existing consul setup with strict reader ACLs.